backend/app/libs/pd_inter_calc.py

```diff
@@ -44,7 +44,7 @@ def fill_until(
         return _current_date
 
     rows = list(hist_transfer_balance.to_dict().items())
-    if len(rows) < 2:  # [FIXME]
+    if len(rows) == 0:
```

Comment:
This case should not happen, the series are always non-empty (see `_get_asset_transfer_balances`)